bpf/custom/bytecount.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+/* Copyright (C) 2020 Authors of Cilium */
+
+struct bpf_elf_map __section_maps bytecount_map = {
+	.type		= BPF_MAP_TYPE_HASH,
+	.size_key	= sizeof(__u32),
+	.size_value	= sizeof(__u64),
+	.max_elem	= 1024,
+};
+
+static __always_inline
+void custom_prog(struct __ctx_buff *ctx, __u32 identity)
```

Comment:
Since the packet isn't supposed to be modified by the custom program, could be mark `ctx` as `const`?

Reply:
No, this would prevent us from using eBPF helpers taking a pointer to the context, even if they're read-only.

Reply:
Hm. Can't we fix the function prototype for those helpers?

Reply:
Maybe. I'll bring this up at tomorrow's datapath meeting I think.